indico/modules/designer/client/js/index.js

```diff
@@ -454,7 +454,7 @@ import {$T} from 'indico/utils/i18n';
           var zIndex = Math.max(selectedItem.zIndex, foremostItem.zIndex + 1);
           div.css('zIndex', zIndex);
           selectedItem.zIndex = zIndex;
-          normalizeZindex();
+          normalizeZIndex();
```

Comment:
Maybe related to the bug we're seeing. Shouldn't we normalize before we set the CSS property?
Something like:
```js
selectedItem.zIndex = zIndex;
normalizeZIndex();
div.css('zIndex', selectedItem.zIndex);
```